contracts/helpers/sol/lib/AdvancedOrderLib.sol

```diff
@@ -365,4 +369,80 @@ library AdvancedOrderLib {
         }
         return orders;
     }
+
+    /**
+     * @dev Converts an AdvancedOrder to a BasicOrderParameters.
+     *
+     * @param advancedOrder  the AdvancedOrder to convert
+     * @param basicOrderType the BasicOrderType to convert to
+     *
+     * @return basicOrderParameters the BasicOrderParameters
+     */
+    function toBasicOrderParameters(
```

Comment:
we may want to consider adding revert logic if an order is unable to be converted, but I'm fine without it for now